po/fr.po

```diff
@@ -29161,80 +29220,81 @@ msgstr "Version de l'arbre source"
 
 #: /tmp/fish/implicit/share/completions/help.fish:1
 msgid "Introduction to the fish syntax"
-msgstr ""
+msgstr "Introduction à la syntaxe de fish"
 
 #: /tmp/fish/implicit/share/completions/help.fish:2
 msgid "Incomplete aspects of fish"
-msgstr ""
+msgstr "Aspects incomplets de fish"
 
 #: /tmp/fish/implicit/share/completions/help.fish:3
 msgid "Known fish bugs"
-msgstr ""
+msgstr "Bogues connus de fish"
 
 #: /tmp/fish/implicit/share/completions/help.fish:4
 msgid "Help on how to reuse previously entered commands"
-msgstr ""
+msgstr "Aide sur la réutilisation des commandes précédentes"
 
 #: /tmp/fish/implicit/share/completions/help.fish:5
 msgid "Help on how tab-completion works"
-msgstr ""
+msgstr "Aide sur l’auto-complétion avec TAB"
 
 #: /tmp/fish/implicit/share/completions/help.fish:6
 msgid "Help on how job control works"
-msgstr ""
+msgstr "Aide sur le contrôle des tâches"
 
 #: /tmp/fish/implicit/share/completions/help.fish:7
 msgid "Summary on how fish differs from other shells"
-msgstr ""
+msgstr "Résumé des différences entre fish et les autres shels"
 
 #: /tmp/fish/implicit/share/completions/help.fish:8
 msgid "Help on how to set the prompt"
-msgstr ""
+msgstr "Aide sur les réglages de l’invite"
 
 #: /tmp/fish/implicit/share/completions/help.fish:9
 msgid "Help on how to set the titlebar message"
-msgstr ""
+msgstr "Aide sur les réglages du titre de la fenêtre"
 
 #: /tmp/fish/implicit/share/completions/help.fish:10
 msgid "Help on how to copy and paste"
-msgstr ""
+msgstr "Aide sur le copier-coller"
 
 #: /tmp/fish/implicit/share/completions/help.fish:11
 msgid "Help on editor shortcuts"
-msgstr ""
+msgstr "Aides sur les raccourcis de l’éditeur"
 
 #: /tmp/fish/implicit/share/completions/help.fish:12
 msgid "Help on environment variables"
-msgstr ""
+msgstr "Aide sur les variables d’environnement"
 
 #: /tmp/fish/implicit/share/completions/help.fish:13
 msgid "Help on setting syntax highlighting colors"
-msgstr ""
+msgstr "Aide sur les couleurs de la coloration syntaxique"
 
 #: /tmp/fish/implicit/share/completions/help.fish:14
 #: /tmp/fish/implicit/share/completions/help.fish:15
 msgid "Help on parameter expansion (Globbing)"
-msgstr ""
+msgstr "Aide sur l’expansion des paramètres (globs)"
 
 #: /tmp/fish/implicit/share/completions/help.fish:17
+#, fuzzy
 msgid "Help on home directory expansion ~USER"
-msgstr ""
+msgstr "Aide sur l’expansion du répertoire utilisateur ~USER"
 
 #: /tmp/fish/implicit/share/completions/help.fish:18
 msgid "Help on brace expansion {a,b,c}"
-msgstr ""
+msgstr "Aide sur l’expansion des accolades {a,b,c}"
 
 #: /tmp/fish/implicit/share/completions/help.fish:19
 msgid "Help on wildcard expansion *.*"
-msgstr ""
+msgstr "Aide sur l’expansion des métacaractères *.*"
 
 #: /tmp/fish/implicit/share/completions/help.fish:20
 msgid "Help on command substitution (SUBCOMMAND)"
-msgstr ""
+msgstr "Aide sur la substitution de commandes"
```

Comment:
“(SUBCOMANND)” intentionally omitted?

Reply:
Was unsure about its use, so stupidly dropped it; corrected.